account_multicompany_code/models/account_account.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, models
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    @api.depends("company_id.code")
+    def _compute_display_name(self):
+        super()._compute_display_name()
+        for record in self:
+            if record.company_id.code:
```

Comment:
I know it was already that way, but since we're changing this, use filtered instead, i.e.:

`for record in self.filtered("company_id.code"):`

Reply:
done